test/e2e/tests/confirmations/transactions/contract-interaction-redesign.spec.ts

```diff
@@ -115,7 +113,7 @@ describe('Confirmation Redesign Contract Interaction Component', function () {
       );
     });
 
-    it(`Opens a contract interaction type 2 transaction that includes layer 1 fees breakdown on a layer 2`, async function () {
+    it.only(`Opens a contract interaction type 2 transaction that includes layer 1 fees breakdown on a layer 2`, async function () {
```

Comment:
```suggestion
    it(`Opens a contract interaction type 2 transaction that includes layer 1 fees breakdown on a layer 2`, async function () {
```

Reply:
awww nice catch!! :heart_eyes:  thank you!